templates/provider/kcm/templates/projectsveltos-crds.yaml

```diff
@@ -1,10 +1,16 @@
 {{- if .Values.projectsveltos.crds.enabled}}
+{{- $labels := include "kcm.labels" . }}
+{{- $annotations := include "kcm.annotations" . }}
 {{- range $path, $_ := .Files.Glob "projectsveltos-crds/*.yaml" }}
 {{- $content := $.Files.Get $path | fromYaml }}
 apiVersion: {{ $content.apiVersion }}
 kind: {{ $content.kind }}
 metadata:
   name: {{ $content.metadata.name }}
+  labels:
+    {{ $labels | nindent 4 }}
```

Comment:
`nindent` is not required here (it is already on a new line) but could be addressed in any other PR